_posts/0000-01-02-kekscreme.md

```diff
@@ -0,0 +1,6 @@
+---
+layout: slide
+title: "Welcome to our second slide!"
+---
```

Comment:
![1pc-PH-Sensor-Module-V1-1-1pc-PH-Probe-for-Uno-R3-board-mega2560-AVR-51-500x500](https://user-images.githubusercontent.com/78673872/107137497-8615d080-690d-11eb-89cd-32c0e5659329.jpg)